language_detector/main.py

```diff
@@ -1,9 +1,25 @@
+# -​*- coding: utf-8 -*​-
+
 """This is the entry point of the program."""
 
 from .languages import LANGUAGES
 
 
 def detect_language(text, languages=LANGUAGES):
-    """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    languages = {'German': 0, 'English': 0, 'Spanish': 0}
+    
+    text_list = text.split()
+    for words in text_list:
+        for dictionaries in LANGUAGES:
+            if words in dictionaries['common_words']:
+                if dictionaries['name'] == 'Spanish':
+                    languages['Spanish'] += 1
```

Comment:
Agreed; this solution does not scale. I have a personal solution I came up with by combining bits from other groups, but it is unclean, to say the least. It does, however, scale.